ops/ccs-ops-misc/load_test/README.md

```diff
@@ -0,0 +1,155 @@
+# BFD Loadtest
+
+## Overview
+The tests in this directory are designed to test the BFD server endpoints when under a certain load of requests, configurable
+using parameters passable when running the tests. These tests use locust, a python-based concurrent test runner that is able to ramp
+requests up to a maximum target number of users making requests against an endpoint. The tests are all written in python, with only a
+few dependencies to install. The tests pull a sample of random data from the target environment's database, attempting to randomize
+to avoid caching affecting the response times of repeat runs.
+
+These tests are intended to be run from a server instance which has access permissions to reach out to the BFD endpoints. How to
+do this is outside the scope of this document, but instructions for this can be found in this more detailed run guide on confluence:
+https://confluence.cms.gov/display/BB2/Run+the+BFD+Load+Tests
+
+
+## Setup
```

Comment:
Wouldn't you need to pip install locust as well?

Reply:
I'm actually not sure, the instances come with some things pre-installed it seems (I didn't need to install python, for instance.) I don't recall if locust was already installed... I'll add a note about this!